lib/dashboard/widget/StrictGraphWidget.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  GraphWidget,
+  HorizontalAnnotation,
+  IMetric,
+  LegendPosition,
+  YAxisProps,
+} from "aws-cdk-lib/aws-cloudwatch";
+
+export interface SingleAxisGraphWidgetProps {
+  readonly title?: string;
+  readonly width: number;
+  readonly height: number;
+  readonly metrics: IMetric[];
+  readonly axis: YAxisProps;
```

Comment:
We may want to make these more explicit, like `leftYAxis`. There's no confusion as to which axis it is then.

Reply:
Option as well. I wanted to remove the notion of side from the props, but it is true that adding the prefix would result in less changes across the lib (no rename would be needed at most places).